sw/device/lib/dif/dif_rv_core_ibex.c

```diff
@@ -309,3 +309,55 @@ dif_result_t dif_rv_core_ibex_read_fpga_info(
                              RV_CORE_IBEX_FPGA_INFO_REG_OFFSET);
   return kDifOk;
 }
+
+dif_result_t dif_rv_core_ibex_get_sw_recov_err_alert(
```

Comment:
This is funny, the doc of this CSR says: "Software recoverable alert. When set to any value other than kMultiBitBool4False, a recoverable alert is sent. Once the alert is sent, the field is then reset to kMultiBitBool4False."

This seems to imply most of the time this will read as False.

Reply:
You are probrably right, but I still wanted to keep this function as the register doc says that it's possible to read it.